decidim-ai/decidim-ai.gemspec

```
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push File.expand_path("lib", __dir__)
+
+require "decidim/ai/version"
+
+Gem::Specification.new do |s|
+  s.version = Decidim::Ai.version
+  s.authors = ["Alexandru-Emil Lupu"]
+  s.email = ["contact@alecslupu.ro"]
+  s.license = "AGPL-3.0"
+  s.homepage = "https://decidim.org"
+  s.metadata = {
+    "bug_tracker_uri" => "https://github.com/decidim/decidim/issues",
+    "documentation_uri" => "https://docs.decidim.org/",
+    "funding_uri" => "https://opencollective.com/decidim",
+    "homepage_uri" => "https://decidim.org",
+    "source_code_uri" => "https://github.com/decidim/decidim"
+  }
+  s.required_ruby_version = "~> 3.3.0"
+
+  s.name = "decidim-ai"
+  s.summary = "A Decidim module with AI tools"
+  s.description = "A module that aims to provide Artificial Intelligence tools for Decidim."
+
+  s.files = Dir["{app,config,db,lib,vendor}/**/*", "Rakefile", "README.md"]
+
+  s.add_dependency "classifier-reborn", "~> 2.3.0"
+  s.add_dependency "cld", "~> 0.11"
+  s.add_dependency "decidim-core", Decidim::Ai.version
```

As we're removing the language detection feature (as it isn't necessary), we should drop the `cld` gem 
```suggestion
  s.add_dependency "decidim-core", Decidim::Ai.version

```

---

done in [6fc7a1a](https://github.com/decidim/decidim/pull/11319/commits/6fc7a1a9792503db7406bb37c56ff1309ff5a972)